portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -22,33 +22,44 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content.*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-    private static final Gson gson = new Gson();
+    private static final Gson GSON = new Gson();
```

Comment:
rename this variable to gson, as it is not a constant

Reply:
Oh so sorry, my bad